bepasty_cli/cli.py

```diff
@@ -95,9 +159,23 @@ def convert(self, value, param, ctx):
     '-i',
     '--insecure',
     help='Disable SSL certificate validation',
-    is_flag=True)
-def main(token, filename, fname, url, ftype, list_pastes, insecure, lifetime):
+    is_flag=True,
+    default=configuration['insecure'])
+@click.option(
+    '--write-config',
+    help='Write current command line arguments and defaults to configuration file '
+         '{} and exit.'.format(CONFIG_FILENAME),
+    is_flag=True,
+    default=False)
+def main(token, filename, fname, url, ftype, list_pastes, insecure, lifetime, write_config):
```

Comment:
add a _-c_ and _--config_ option so people can define the configfile location themselves, fallback on defaults if none is configured.